website/views.py

```
@@ -5,10 +5,21 @@
+from .models import Record
+from django.core.paginator import Paginator
 
 # Create your views here.
 
 def home(request):
+    try:
+        records_list = Record.objects.all().order_by('id')
+        paginator = Paginator(records_list, 10)
+        page = request.GET.get('page')
+        records = paginator.get_page(page)
+    except Exception as e:
```

Ruff (0.8.0)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)